apps/design-land/src/app/app.component.html

```diff
@@ -1,5 +1,9 @@
-<daff-sidebar-viewport>
-  <daff-sidebar class="design-land__sidebar">
+<nav daff-navbar>
+  <div>Design Land</div>
+  <design-land-theme-switch class="design-land__theme-switch"></design-land-theme-switch>
```

Comment:
```suggestion
  <daff-theme-switch-button class="design-land__theme-switch"></daff-theme-switch-button>
```

Reply:
Can't add a comment to line 48, but that line can be removed in favor of this section.

Reply:
I didn't understand what this was referring to.

Reply:
Line 48 of this file also uses this component: `<daff-theme-switch-button class="design-land__theme-switch"></daff-theme-switch-button>`

It should be removed there because the theme switch button goes in the nav.

Reply:
Ah, I see. Resolved in b3e5ad1a